apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -205,9 +205,13 @@ export class SendMessageEmail extends SendMessageBase {
       to: email,
       subject,
       html,
-      from: command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
+      from:
+        overrides?.email?.from || command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
```

Comment:
```suggestion
        overrides?.email?.from || integration?.credentials.from || 'no-reply@novu.co',
```

$sender_email can be deprecated we don't use it since we move to the new novu system from stateless, so i think we can remove this.